nx-dev/feature-doc-viewer/src/lib/related-documents-section.tsx

```diff
@@ -53,7 +53,7 @@ function CategoryBox({ category }: { category: RelatedDocumentsCategory }) {
           >
             <Link
               href={d.path}
-              className="flex flex-grow items-center justify-between no-underline hover:text-sky-600 hover:underline dark:hover:text-sky-400"
+              className="flex flex-grow items-center justify-between  no-underline transition-colors ease-out hover:text-blue-700 hover:underline dark:text-sky-500 dark:hover:text-sky-400 "
```

Comment:
You have extra spaces, can you run prettier? It should have already taken out these ones normally.

Reply:
dang, I had corrected a bunch of those. Also thought prettier would catch them, but apparently doesn't 🤔 . Pushing a fix